src/appleseed.shaders/src/maya/as_maya_areaLight.osl

```diff
@@ -106,6 +106,19 @@ surface as_maya_areaLight
     ]],
 )
 {
+    out_center = point("center", 0, 0, 0);
```

Comment:
@dictoon it should actually be *shader* space, but there is no *shader* space anymore, so the best we can do is to use *object* space, and center will be (0,0,0) of that space. Addressed with 2cfe7f799ec1a8c9fc785af6c52bfc1b252bb62e